bigbluebutton-tests/playwright/webcam/util.js

```diff
@@ -11,23 +15,44 @@ async function webcamContentCheck(test) {
     console.log(`loop ${i}`);
     const checkCameras = () => {
       const videos = document.querySelectorAll('video');
-      const lastVideoColor = document.lastVideoColor || {};
-      document.lastVideoColor = lastVideoColor;
+      const lastVideoHash = document.lastVideoHash || {};
+      document.lastVideoHash = lastVideoHash;
+
+      // If this code was running in the playwright test harness, we
+      // could just require('sha1'), but it's running in the test
+      // browser via test.page.evaluate, and I'm not sure how to get a
+      // node module (like sha1) into the test browser.  So I just
+      // found a nice hash function on the Internet.  -bwb
+
+      // public domain code from https://stackoverflow.com/a/52171480/1493790
+      // and/or https://github.com/bryc/code/blob/master/jshash/experimental/cyrb53.js
+
+      const cyrb53 = function(array, seed = 0) {
```

Comment:
If you required sha1 at the top of this file and assign sha1 function to a local variable here, would it work?

Reply:
Good question.  I don't think I ever figured out how to import a function in the test harness and then use it in the test browser.  I'll look at it again.

Reply:
Thanks for updating the function to the one from Web Crypto API!